plugins/JetAnalyzer.cc

```diff
@@ -1144,7 +1184,7 @@ float JetAnalyzer::CalculateHT(const edm::Event& iEvent, const edm::EventSetup&
 
     //Eta and pt thresholds most commonly used for HT
     float HT = 0.;
-    int JetId = id;//common setting: 3, tight jets
+    int JetId = id;//common setting: 2, tight jets
```

Comment:
Minor comment: isn't JetId 2 equal to medium WP then?

Reply:
Medium WP is no longer supported (it's a 2015 remnant)

Reply:
Just to be sure: We are anyways using the `isTightJet` function and the `JetId` value was only used "internally" in our framework to choose the quality cut, right? And since we were being consistent, I think this shouldn't really be a problem

On the other hand, there might be a related issue to keep in mind: Looking at [1], it seems like our `isTightJet` function is not up to date. @lbenato, could you please take a look?

[1] https://twiki.cern.ch/twiki/bin/viewauth/CMS/JetID